src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px $header-shadow-color;
```

Comment:
Ensure that the variable `$header-shadow-color` is defined in `variables.scss` and that the color meets the design requirements.